antsibull/data/docsite/role.rst.j2

```diff
@@ -0,0 +1,286 @@
+.. Document meta
+
+:orphan:
+
+{# If we can put together source and github repo, we could make the Edit me of github button work.
+   See meta.get("source") in Ansible's docs/docsite/_themes/sphinx_rtd_theme/breadcrumbs.html
+   for more information
+:source: @{ source }@
+-#}
+
+.. Anchors
+
+.. _ansible_collections.@{plugin_name}@_@{plugin_type}@:
+
+.. Anchors: aliases
+
+{# TODO: This assumes that aliases will be short names.
+   If they're FQCN, then we need to change this
+#}
+
+.. Title
+
+{% if entry_points.main and entry_points.main.short_description -%}
+{%   set title = plugin_name + ' -- ' + entry_points.main.short_description | rst_ify -%}
+{% else -%}
+{%   set title = plugin_name -%}
+{% endif -%}
+
+@{ title }@
+@{ '+' * title|length }@
+
+.. Collection note
+
+{% if collection == 'ansible.builtin' -%}
+.. note::
+    This role is part of ``ansible-core`` and included in all Ansible
+    installations.
+{% else %}
+.. note::
+    This role is part of the `@{collection}@ collection <https://galaxy.ansible.com/@{collection | replace('.', '/', 1)}@>`_{% if collection_version %} (version @{ collection_version }@){% endif %}.
+
+    To install it use: :code:`ansible-galaxy collection install @{collection}@`.
+
+    To use it in a playbook, specify: :code:`@{plugin_name}@`.
+{% endif %}
+
+.. contents::
+   :local:
+   :depth: 2
+
+{% for entry_point, ep_doc in entry_points.items() | sort %}
+
+.. Entry point title
+
+{%   if ep_doc['short_description'] -%}
+{%     set title = 'Entry point ``' + entry_point + '`` -- ' + ep_doc['short_description'] | rst_ify -%}
+{%   else -%}
+{%     set title = 'Entry point ``' + entry_point + '``' -%}
+{%   endif -%}
+
+@{ title }@
+@{ '-' * title|length }@
+
+.. version_added
```

Comment:
Maybe add a label here to clarify that it's the collection version?

Reply:
Implemented in 71dd24778f3ef143152432bbcc7b15f2bd0026f1.